src/android/jk/cordova/plugin/kiosk/KioskActivity.java

```diff
@@ -78,12 +82,13 @@ protected void onActivityResult(int requestCode, int resultCode,  Intent data) {
             SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(this.getApplicationContext());
             sp.edit().putBoolean(PREF_KIOSK_MODE, true).commit();
             if (Settings.canDrawOverlays(this)) {
-                addOverlay();
+                allowImmersiveSystemUiOverlay = true;
             }
         }
     }
+
     //http://stackoverflow.com/questions/25284233/prevent-status-bar-for-appearing-android-modified?answertab=active#tab-top
-    public void addOverlay() {
+    private void addOverlay() {
```

Comment:
@kptdobe That's why I changed it so it's clear that this method is actually only used "privately". There was no reason to expose it in the first place to either other Activities or via JS.